LavalinkServer/src/main/java/lavalink/server/player/AudioLoaderRestHandler.kt

```
@@ -0,0 +1,175 @@
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package lavalink.server.player
+
+import com.sedmelluq.discord.lavaplayer.player.AudioPlayerManager
+import com.sedmelluq.discord.lavaplayer.track.AudioTrack
+import dev.arbjerg.lavalink.api.AudioLoaderJsonModifier
+import lavalink.server.config.ServerConfig
+import lavalink.server.util.Util
+import org.json.JSONArray
+import org.json.JSONObject
+import org.slf4j.LoggerFactory
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import java.io.IOException
+import java.util.concurrent.CompletionStage
+import javax.servlet.http.HttpServletRequest
+
+@RestController
+class AudioLoaderRestHandler(
+    private val audioPlayerManager: AudioPlayerManager,
+    private val serverConfig: ServerConfig,
+    private val loaderExtensions: List<AudioLoaderJsonModifier>
+) {
+    companion object {
+        private val log = LoggerFactory.getLogger(AudioLoaderRestHandler::class.java)
+
+        private inline fun <reified T> Any?.intoOrNull(): T? = this as? T
+
+        private fun log(request: HttpServletRequest) = log.info("GET ${request.servletPath}")
+
+        private fun attemptModifier(modifier: AudioLoaderJsonModifier, json: JSONObject, block: AudioLoaderJsonModifier.(JSONObject) -> JSONObject): JSONObject? {
+            return json.clone()
+                .runCatching { modifier.block(this) }
```

This looks like it will swallow the error without logging it. A simple try/catch would probably be a better and simpler alternative to this method